x-pack/plugins/maps/common/constants.ts

```diff
@@ -233,3 +233,15 @@ export enum LAYER_WIZARD_CATEGORY {
   REFERENCE = 'REFERENCE',
   SOLUTIONS = 'SOLUTIONS',
 }
+
+export enum VECTOR_SHAPE_TYPE {
+  POINT = 'POINT',
+  LINE = 'LINE',
+  POLYGON = 'POLYGON',
+}
+
+// https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/#data-expressions
+export enum MB_LOOKUP_FUNCTION {
```

Comment:
Consider exporting as string constant  `'get' | 'feature-state'`.

If this is an enumeration, I feel like we should use it consistently:
- in the `match` and `to-string` expressions of `dynamic_*_property`.
- in the `geometry-type` in `mb_filter_expression`.